checker/src/main/java/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java

```diff
@@ -175,7 +176,9 @@ public void addComputedTypeAnnotations(Tree tree, AnnotatedTypeMirror type, bool
         // If dataflow shouldn't be used to compute this type, then do not use the result from
         // the Value Checker, because dataflow is used to compute that type.  (Without this,
         // "int i = 1; --i;" fails.)
-        if (iUseFlow && tree != null && TreeUtils.isExpressionTree(tree)) {
+        if (tree != null
+                && TreeUtils.isExpressionTree(tree)
+                && (iUseFlow || tree instanceof LiteralTree)) {
```

Comment:
Without this, the interaction between the Initialized Fields Checker and the Index Checker fails, because the Initialized Fields Checker calls this (transitively) with iUseFlow set to false.

Literals don't depend on flow, so this change seems safe.